backend/tasks.py

```diff
@@ -132,6 +132,37 @@ def save_UI_state(*args, **kwargs):
         utils.commit_with_retry(transaction_session)
 
     return 200 # success
+# Create_child   
+# transaction_session, db = utils.create_transaction_session() [DONE]
+# document = db.documents.find_one({"_id": document_id}) [DONE]
+# document["text"][s:e]
+# Declare any fields that are necessary
+# Need the vectorize function # Can write a test for this one -> check if it's actually working
+# inserted_document = db.documents.insert_one( ... )
+# inserted_document.insertedID
+# id = insertedid
+# update_one(…)
+# Write a test to check if it's actually working (can insert a dummy document and delete it once it's done)
+@app.task
+def create_child(*args, **kwargs):
+    transaction_session, db = utils.create_transaction_session()
+    with transaction_session.start_transaction(): 
+        document_id = kwargs["document_id"]
```

Comment:
One style we seem to be converging on is doing the conversion right away, i.e.:

`document_id = ObjectId(str(kwargs["document_id"]))`